src/ORM/Marshaller.php

```diff
@@ -105,6 +105,18 @@ public function one(array $data, array $options = [])
         $propertyMap = $this->_buildPropertyMap($options);
 
         $schema = $this->_table->schema();
+        $primaryKey = $schema->primaryKey();
+
+        if ( array_intersect($primaryKey, array_keys($data)) == $primaryKey ) {
```

Comment:
Won't this always pass when the primary key is present? Also what about associated values that should be marshaled under this entity?